app/services/slack_notification_service.rb

```diff
@@ -0,0 +1,27 @@
+class SlackNotificationService
+  class << self
+    def post(message)
+      new.post(message)
+    end
+  end
+
+  def post(message)
+    return unless slack_configured?
+
+    Slack.chat_postMessage text: message, username: "PoemMaster", channel: slack_channel
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Line is too long. [88/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)